prow/config/jobs/cni.yaml

```diff
@@ -13,7 +13,10 @@ jobs:
     command: [make, lint]
   - name: cni-install
     command: [make, docker, test]
-
+  - name: cni-e2e
```

Comment:
Can we have something that ensures the generated version is up to date?

Reply:
yes we do have a presubmit check (generate-check)